requirements.txt

```diff
@@ -0,0 +1,15 @@
+astroid==2.2.5
```

Comment:
Ah, yes, quite useful. Looks like this is a tip learnt from the [Python tutorial](https://docs.python.org/3/tutorial/venv.html#managing-packages-with-pip)